client/plots/DEanalysis.js

```diff
@@ -53,6 +53,24 @@ class DEanalysis {
 	async setControls(output) {
 		this.dom.controlsDiv.selectAll('*').remove()
 		const inputs = [
+			{
+				label: 'Min count',
+				type: 'number',
+				chartType: 'DEanalysis',
+				settingsKey: 'min_count',
+				title: 'Min count',
```

Comment:
at title, fully specify the purpose "Genes with xxx will be excluded"

Reply:
I added a title to both min_count and min_total_count.